src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import  booksReducer  from './books/books';
+
+const reducer = combineReducers({
+  booksReducer
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger)
+);
+
+export default store; 
```

Comment:
- [ ] The categories.js file is empty. Following the project requirements, you should write your categories actions and reducer in `/src/redux/categories/categories.js`. Check out the requirements below 👇:

<img width="623" alt="image" src="https://user-images.githubusercontent.com/77190203/167258724-2140e15a-a1a2-49eb-b10e-83637933d03e.png">

Reply:
Thank you for pointing out! I was in a hurry and didn't even notice that I didn't commit this as well. Thank you very much!